php.nanorc

```diff
@@ -3,19 +3,22 @@ syntax "PHP" "\.php[2345s~]?$|\.module$"
 header "^#!.*php"
 magic "PHP script"
 comment "//"
+
+# Default
 color white start="<\?(php|=)?" end="\?>"
 # Constructs
 color brightblue "(class|extends|goto) ([a-zA-Z0-9_]*)"
-color brightblue "[^a-z0-9_-]{1}(var|class|function|echo|case|break|default|exit|switch|if|else|elseif|endif|foreach|endforeach|@|while|public|private|protected|return|const|static|extends|as|array|require|include|require_once|include_once|define|do|continue|declare|goto|print|in|namespace|use)[^a-z0-9_-]{1}"
+color brightblue "\<(var|class|function|echo|case|break|default|exit|switch|if|else|elseif|endif|foreach|endforeach|@|while|public|private|protected|return|const|static|extends|as|array|require|include|require_once|include_once|define|do|continue|declare|goto|print|in|namespace|use)\>"
 color brightblue "[a-zA-Z0-9_]+:"
 # Variables
-color green "\$[a-zA-Z_0-9$]*|[=!<>]"
-color green "\->[a-zA-Z_0-9$]*|[=!<>]"
```

Comment:
1. Notice here that I removed `$` from `[a-zA-Z_0-9$]`, because I'm not sure why do the author think that there can be `$` within the variable name? To my understandings, PHP doesn't allow variable like `$nam$e`.

2. I also removed `[=!<>]`, because I not sure why can `=`, `!`, `<` and `>` symbols represent a variable in PHP?